osu.Game/Screens/OnlinePlay/Lounge/Components/DrawableRoom.cs

```diff
@@ -304,38 +307,83 @@ private void load()
             }
         }
 
-        private class RoomHostText : OnlinePlayComposite
+        private class RoomStatusText : OnlinePlayComposite
         {
-            private LinkFlowContainer hostText;
+            [Resolved]
+            private OsuColour colours { get; set; }
+
+            private SpriteText statusText;
+            private LinkFlowContainer beatmapText;
 
-            public RoomHostText()
+            public RoomStatusText()
             {
-                AutoSizeAxes = Axes.Both;
+                RelativeSizeAxes = Axes.X;
+                AutoSizeAxes = Axes.Y;
+                Width = 0.5f;
             }
 
             [BackgroundDependencyLoader]
             private void load()
             {
-                InternalChild = hostText = new LinkFlowContainer(s => s.Font = OsuFont.GetFont(size: 16))
+                InternalChild = new GridContainer
                 {
-                    AutoSizeAxes = Axes.Both
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    ColumnDimensions = new[]
+                    {
+                        new Dimension(GridSizeMode.AutoSize),
+                    },
+                    Content = new[]
+                    {
+                        new Drawable[]
+                        {
+                            statusText = new OsuSpriteText
+                            {
+                                Font = OsuFont.Default.With(size: 16),
+                                Colour = colours.Lime1
+                            },
+                            beatmapText = new LinkFlowContainer(s =>
+                            {
+                                s.Font = OsuFont.Default.With(size: 16);
+                                s.Colour = colours.Lime1;
+                            })
+                            {
+                                RelativeSizeAxes = Axes.X,
+                                AutoSizeAxes = Axes.Y
+                            }
+                        }
+                    }
                 };
             }
 
             protected override void LoadComplete()
             {
                 base.LoadComplete();
+                SelectedItem.BindValueChanged(onSelectedItemChanged, true);
+            }
+
+            private void onSelectedItemChanged(ValueChangedEvent<PlaylistItem> item)
+            {
+                beatmapText.Clear();
 
-                Host.BindValueChanged(host =>
+                if (Type.Value == MatchType.Playlists)
                 {
-                    hostText.Clear();
+                    statusText.Text = "Waiting for players";
+                    return;
+                }
```

Comment:
if this text is to be shown on every single playlist in the playlists lounge regardless of the room's playlist, does it need to be shown at all?

Reply:
The alternative is to move the title downwards, at the cost of there being some gigantic waste of space. I consulted with flyte and he prefers that _something_ is always shown.

Reply:
Should probably read something like "Ready to play" in that case?

Reply:
Sure.